DoSomething/gladiator
# Modeling on the laravel runway

References #2 

#### What's this PR do?
This PR updates the database schema and models. It also allows Users and WaitingRooms to be associated via pivot tables properly and includes some Model Factories and Seeds to generate 2 Admin Users, 3 WaitingRooms, 300 Contestant Users that get divvied out into any of the 3 rooms and associated via the pivot table.

#### Where should the reviewer start?
Probably best to start on the migrations, then move to the models and lastly the seeds and model factory.

#### How should this be manually tested?
Pull down code, then run: 
```
$ composer dump-autoload
$ php artisan migrate
$ php artisan db:seed
```

And check if you get a bunch of users in the database associated with waiting rooms, along with 2 admin users.
 
#### What are the relevant tickets?
#2

---
@angaither @sbsmith86 @deadlybutter 

cc: @DFurnes
